en/restore-data-using-tidb-lightning.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Restore Data into TiDB in Kubernetes
-summary: Learn how to quickly restore data into a TiDB cluster in Kubernetes with TiDB Lightning.
+title: Import Data
```

Comment:
This doc only describes how to deploy TiDB Lightning, but not any mention of how to use it or how to import data.

Reply:
After TiDB Lightning is deployed, a data import jos will start. so we deploy TiDB Lightning to import data.

Reply:
Got it. Thanks. 
Will you just tell users that as well? They may have the same confusion as I did.